public/index.html

```diff
@@ -1 +1,12 @@
-<div id="root"></div>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>metastudent.se</title>
```

Comment:
```suggestion
    <title>META Student</title>
```
or similar, not sure what would be best